arviz/plots/ecdfplot.py

```diff
@@ -28,40 +28,47 @@ def plot_ecdf(
     backend_kwargs=None,
     **kwargs
 ):
-    """Plot ECDF or ECDF-Difference Plot with Confidence bands.
+    r"""Plot ECDF or ECDF-Difference Plot with Confidence bands.
 
-    This plot uses the simulated based algorithm presented in the paper "Graphical Test for
-    Discrete Uniformity and its Applications in Goodness of Fit Evaluation and
-    Multiple Sample Comparison" [1]_.
+    Plots of the empirical CDF estimates of an array. When `values2` option is different from
+    `None`, the two empirical CDFs are overlaid with the distribution of `values` on top
+    (in a darker shade) and confidence bands in a more transparent shade. Optionally, the difference
+    between the two empirical CDFs can be computed, and the PIT for a single dataset or a comparison
+    between two samples.
+
+    Notes
+    -----
+    This plot computes the confidence bands with the simulated based algorithm presented in [1]_.
 
     Parameters
     ----------
     values : array-like
-        Values to plot from an unknown continuous or discrete distribution
+        Values to plot from an unknown continuous or discrete distribution.
     values2 : array-like, optional
-        Values to compare to the original sample
-    cdf : function, optional
-        Cumulative distribution function of the distribution to compare the original sample to
-    difference : bool, optional, Defaults False
-        If true then plot ECDF-difference plot otherwise ECDF plot
-    pit : bool, optional
-        If True plots the ECDF or ECDF-diff of PIT of sample
-    confidence_bands : bool, optional, Defaults True
-        If True plots the simultaneous or pointwise confidence bands with 1 - fpr confidence level
-    pointwise : bool, optional, Defaults False
-        If True plots pointwise confidence bands otherwise simultaneous bands
-    npoints : int, optional, Defaults 100
-        This denotes the granularity size of our plot
-        i.e the number of evaluation points for our ecdf or ecdf-difference plot
-    num_trials : int, optional, Defaults 500
-        The number of random ECDFs to generate to construct simultaneous confidence bands
-    fpr : float, optional, Defaults 0.05
-        The type I error rate s.t 1 - fpr denotes the confidence level of bands
-    figsize : tuple, optional
-        Figure size. If None it will be defined automatically.
-    fill_band : bool, optional
-        Use fill_between to mark the area inside the credible interval.
-        Otherwise, plot the border lines.
+        Values to compare to the original sample.
+    cdf : function, optional, default None
```

Comment:
```suggestion
    cdf : callable, optional
```

I think the default is not comparing to any cdf, I think optional is a better fit, in any case, optional and default can't be used at the same time.